keyboards/polilla/rev1/rules.mk

```diff
@@ -0,0 +1,14 @@
+# MCU name
+MCU = STM32F042
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+LTO_ENABLE = yes
```

Comment:
LTO has some weird behavior on ARM.  You may not want to have it enabled. 

```suggestion
```

Reply:
Thanks for your comments, I implemented your suggestions.